packages/xsel/build.sh

```diff
@@ -0,0 +1,29 @@
+TERMUX_PKG_HOMEPAGE=http://www.kfish.org/software/xsel/
+TERMUX_PKG_DESCRIPTION="Command-line program for getting and setting the contents of the X selection"
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_MAINTAINER="Doron Behar <me@doronbehar.com>"
+TERMUX_PKG_VERSION=1.2.0.20190505
+_commit=24bee9c7f4dc887eabb2783f21cbf9734d723d72
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_DEPENDS="libx11"
+TERMUX_PKG_BUILD_DEPENDS="libxt"
+
+termux_step_extract_package() {
+	mkdir -p $TERMUX_PKG_SRCDIR
+	cd $TERMUX_PKG_SRCDIR
+	git clone https://github.com/kfish/xsel
+	cd xsel
+	git checkout $_commit
+	./autogen.sh --prefix=$TERMUX_PREFIX
+}
+
+termux_step_make() {
+	cd $TERMUX_PKG_SRCDIR/xsel
+	make
+}
+
+termux_step_make_install() {
+	cd $TERMUX_PKG_SRCDIR/xsel
+	make install
+	install -D -m0644 COPYING $TERMUX_PREFIX/share/LICENSES/xsel.txt
```

Comment:
That why you got:
```
ERROR: No LICENSE file was installed for xsel
```

Installation of these files handled automatically in most cases by hook: https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh.
If needed manual, then just specify file in `TERMUX_PKG_LICENSE_FILE`.

But for common licenses like `MIT` just set it's name.

Reply:
License fixeby myself.